src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,94 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { Person } from '../../types';
+
+type Props = {
+  people: Person[];
+  slug: string;
+};
+
+export const PeopleTable: React.FC<Props> = ({ people, slug }) => {
+  const personRowRender = (person: Person) => {
+    const findPerson = (personName: string) => {
+      const someone = people.find(somebody => somebody.name === personName);
+
+      if (someone) {
+        return (
+          <Link
```

Comment:
Here you should instead create component PersonLink in which you'll pass person according to the task